src/test/groovy/sample/SampleJobSpec.groovy

```diff
@@ -0,0 +1,93 @@
+package sample
+
+import groovy.util.slurpersupport.GPathResult
+import groovy.util.slurpersupport.Node
+import javaposse.jobdsl.dsl.DslScriptLoader
+import javaposse.jobdsl.dsl.JobManagement
+import javaposse.jobdsl.plugin.JenkinsJobManagement
+import javaposse.jobdsl.dsl.GeneratedItems
+import javaposse.jobdsl.dsl.GeneratedJob
+import org.junit.ClassRule
+import org.jvnet.hudson.test.JenkinsRule
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+
+/**
+ * Basic POC tests for the sample dsl script sample/jobs/sampleJob.groovy
+ */
+class SampleJobSpec extends Specification {
+
+    @Shared
+    @ClassRule
+    JenkinsRule jenkinsRule = new JenkinsRule()
+
+    @Unroll
+    void 'test no exceptions thrown when dsl is run'() {
+
+        given:
+        JobManagement jm = new JenkinsJobManagement(System.out, [:], new File('.'))
+        DslScriptLoader loader = new DslScriptLoader(jm)
+        File dslScript = new File('sample/jobs/sampleJob.groovy')
+
+        when:
+        loader.runScript(dslScript.text)
+
+        then:
+        noExceptionThrown()
```

Comment:
Write a comment saying that these are the assertions, for anyone who doesn't know groovy it is really confusing.